Meadowlark-js/backends/meadowlark-opensearch-backend/src/repository/QueryOpensearch.ts

```diff
@@ -38,8 +38,8 @@ function sanitize(input: string): string {
  * Convert query string parameters from http request to OpenSearch
  * SQL WHERE conditions. Returns empty string if there are none.
  */
-function whereConditionsFrom(queryStringParameters: object): string {
-  return Object.entries(queryStringParameters)
+function whereConditionsFrom(queryParameters: object): string {
+  return Object.entries(queryParameters)
```

Comment:
Renamed to drop unnecessary "string" from name to be consistent with dropping it elsewhere to shorten things up.